tests/prague/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -623,6 +699,16 @@ def test_wide_container(eof_test: EOFTestFiller, width: int, exception: EOFExcep
             ),
             id="EOF1_eofcreate_valid_2",
         ),
+        pytest.param(
+            Container(
+                sections=[Section.Code(Op.PUSH1[0] + Op.RJUMP[0] + Op.STOP)]
+                + 256 * [Section.Container(Container(sections=[Section.Code(Op.INVALID)]))],
```

Comment:
`Container(sections=[Section.Code(Op.INVALID)])` can be abbreviated to `Container.Code(Op.INVALID)`

Reply:
changed.